.github/workflows/phpunit.yml

```diff
@@ -0,0 +1,35 @@
+name: PHPUnit
+
+on:
+  pull_request:
+  push:
```

Comment:
Wouldn't that combination run the pipeline twice if a PR gets a push? Once for the `pull request synchronize` event, and once for the `push` event?

Reply:
Yes. Guess they would run twice in that case. But as the runtime of the tests is very fast, I don't think that's a big problem atm. If we ever start to make bigger changes on that component again, I guess we can consider changing that again if needed. But for now I'm tempted to ignore it...